README.md

```diff
@@ -34,6 +34,9 @@ If you'd like to join, just go through the [issues list](https://github.com/next
 
 ## 🏗 Development setup
 
+This app requires the main branch of the [Viewer app](https://github.com/nextcloud/viewer) to be installed and enabled.
```

Comment:
Depending on the branch you are working on, you might need a `stableXX` branch. Maybe we can just skip the detail about the branch, what do you think?

```suggestion
This app requires the [Viewer app](https://github.com/nextcloud/viewer) to be installed and enabled.
```

Reply:
Nice, yes I think it is important to mention this :+1: